gp-templates/translations.php

```diff
@@ -32,6 +32,7 @@
 		<option value="" selected="selected"><?php _e( 'Bulk Actions', 'glotpress' ); ?></option>
 		<option value="approve"><?php _e( 'Approve', 'glotpress' ); ?></option>
 		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
+		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
```

Comment:
I don't disagree but if we change one we should change them all and I think it will be obvious enough that "Fuzzy" is an action in this case (it is in the "Bulk Actions" drop down after all).

We use fuzzy as an action when importing as well.